@shared/api/helpers/getIconUrlFromIssuer.ts

```diff
@@ -42,7 +42,7 @@ export const getIconUrlFromIssuer = async ({
   try {
     /* First, check our localStorage cache in Background to see if we've found this url before */
     ({ iconUrl } = await sendMessageToBackground({
-      assetCode: code,
+      assetCanonical: `${code}:${key}`,
```

Comment:
Does the canonical form here handle the case where the code is an ASSET_TYPE_ALPHANUM12 but only contains 3 non-zero bytes?

Reply:
could you give an example of what you mean by `only contains 3 non-zero bytes`?

Reply:
I think it's a reference to this subtlety in SEP-11, https://stellar.org/protocol/sep-11#alphanum4-alphanum12:

> Stellar disallows assets of type ASSET_TYPE_CREDIT_ALPHANUM12 that have fewer than 5 bytes, but such assets can be represented in binary XDR, and so in txrep an AlphaNum12 value with fewer than 5 bytes in its assetCode is rendered with trailing \x00 (escaped NUL bytes) to as to make the length 5---e.g., the 12-byte asset code ABC is rendered ABC\x00\x00. Note that stellar-core disallows non ASCII bytes in AssetCode fields, so the primary use of this feature is to construct or examine invalid transaction test cases.

Reply:
On re-read, I think it is not relevant because of:
>Note that stellar-core disallows non ASCII bytes in AssetCode fields, so the primary use of this feature is to construct or examine invalid transaction test cases.